packages/pn-commons/src/components/NotificationDetail/NotificationPaymentF24Item.tsx

```diff
@@ -186,7 +186,7 @@ const NotificationPaymentF24Item: React.FC<Props> = ({
           </Typography>
         ) : (
           <>
-            <Typography variant="sidenav" color="text.primary">
+            <Typography variant="sidenav" color="text.primary" sx={{ overflowWrap: 'anywhere' }}>
```

Comment:
Fix is good but it doesn't fit all use cases. We can try to do something different.

Into getElement replace Box with Stack (display="flex" becomes useless)

The Box element after the main return can be replaced with Stack (display="flex" and alignSelf="stretch" become useless). The f24 title and the button must be on the same row in the desktop view and on different rows in the mobile view. In the mobile view the button must be left centered. The title and the button must be spaced by 8px.
TIPS: this can be accomplished using direction, alignItems and spacing properties.

Replace the first Box inside the main one with Stack (display="flex" becomes useless, flexDirection="column" can be replaced with direction="column" and flex="1 0 0" can be replaced with flexGrow="1").

Replace the Box into the maxTimeError condition with Stack (display="flex" becomes useless)

Remove Box that wrap getElement().

When you have done this changes CHECK ALL the component layouts (initial, dowload in progress, download that takes much time, download in error, F24 linked to a pagoPa payment and so on) in desktop view and mobile view.